src/libs/NextStepUtils.ts

```diff
@@ -121,7 +116,14 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
             if (harvesting?.enabled && autoReportingFrequency !== CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MANUAL) {
                 optimisticNextStep.message = [
                     {
-                        text: 'These expenses are scheduled to ',
+                        text: 'Waiting for ',
+                    },
+                    {
+                        text: `${ownerDisplayName} `,
+                        type: 'strong',
+                    },
+                    {
+                        text: 'expense(s) to',
```

Comment:
I think we should keep the `%expenses` here and then update the parsing logic 

https://github.com/Expensify/App/blob/f66caf4a55d32908b96fcd763332660ac8ba64ed/src/libs/NextStepUtils.ts#L55-L59 so we render the correct format as per the doc